html/browsers/origin/cross-origin-objects/cross-origin-objects-exceptions.html

```diff
@@ -71,8 +71,12 @@
  * Also tests for [[GetOwnProperty]] and [[HasOwnProperty]] behavior.
  */
 
+var whitelistedWindowIndices = ['0', '1'];
 var whitelistedWindowProps = ['location', 'postMessage', 'window', 'frames', 'self', 'top', 'parent',
                               'opener', 'closed', 'close', 'blur', 'focus', 'length'];
+whitelistedWindowProps = whitelistedWindowProps.concat(whitelistedWindowIndices);
```

Comment:
OK, so this includes the indices, but not the symbols, right?  We should really sync these tests against each other; the only differences between them should the the assert_throws lines, where cross_origin_objects has `null` as the first arg, and cross-origin-objects-exceptions has `"SecurityError"`...   That's what we should do in the followup.

Thank you for catching this!